uniflow/op/extract/split/markdown_header_splitter.py

```diff
@@ -54,6 +56,7 @@ def __call__(
             value_dict = copy.deepcopy(node.value_dict)
             text = value_dict["text"]
             text = self.header_splitter(text.strip(), headers_to_split_on_list)
+            text = self.format_splitter(text)
```

Comment:
qq: is this a bug fix for the previous implementation which does not `format_splitter`'s result? Is there any other places using `MarkDownHeaderSplitter` in which this change will have a side effect?